addon/components/frost-table-cell-renderer.js

```diff
@@ -14,8 +14,12 @@ export default Component.extend({
   // == PropTypes =============================================================
 
   propTypes: {
-    // options
+    // optional
+    colIndex: PropTypes.number,
```

Comment:
I'm  trying to understand why this is needed. Why would the individual cell renderer need to know its column index? The index will already be in the callback from the closure, right? 

Also, if it is needed, it should probably be either required, or have a default value provided.